packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncSession.kt

```diff
@@ -73,6 +74,21 @@ public interface SyncSession {
      */
     public val state: State
 
+    /**
+     * The [SyncConfiguration] responsible for controlling the session.
+     */
+    public val configuration: SyncConfiguration
+
+    /**
+     * The [User] defined by the [SyncConfiguration] to authenticate the session on Atlas App Services.
+     */
+    public val user: User
+
+    /**
+     * The URL describing the remote Realm that this session connects to and synchronizes changes with.
+     */
+    public val serverUrl: String
```

Comment:
Not sure we should be exposing this - for pretty much all users, this will be `realm.mongodb.com` and the internal users that absolutely need to look it up can go through the user.

Reply:
Good point. This is probably a leftover from the pre-v10 API's. But you are correct, there is no reason we need to expose this in Kotlin. I just forgot to think more closely about it when creating this issue.

Reply:
Ah okay! I'll revert that one 👍

Reply:
The only use case I can think of is testing, If we spin up a local BAAS?